### DISCUSSION TITLE
:c instead of  :bc

### DISCUSSION THREAD

**Discussion by aspizu** [helix-editor/helix#5582] [Ideas]

**Comment by the-mikedavis** [comment]
It would be possible to remap this yourself with https://github.com/helix-editor/helix/issues/4423. For now you can edit the alias in the source if you're building from source: https://github.com/helix-editor/helix/blob/3cf5216dbd1b9c767e6a922f93f1a3b1a2955d02/helix-term/src/commands/typed.rs#L1852

I don't think this is a good default though. `:c` could be used for some of the other typable commands that start with 'c' like `:config-open` or `:cd` or `:cquit` depending on how much one uses any of these commands.

**Comment by aspizu** [comment]
What about?:  `:q` closing the current buffer, and exiting helix if there are no buffers open.

**Comment by the-mikedavis** [comment]
AFAIK it isn't possible to make a command like that with the current set of commands. It seems like a rather niche workflow to me though so I would prefer that something like that is implemented later when there is a way to script commands or create them via plugins
